Formula/jack.rb

```diff
@@ -24,8 +24,16 @@ class Jack < Formula
   depends_on "libsndfile"
   depends_on "libsamplerate"
 
-  # Change pThread header include from CarbonCore
-  patch :p0, :DATA if MacOS.version >= :mountain_lion
+  def install
+    # Makefile hardcodes Carbon header location
+    inreplace Dir["drivers/coreaudio/Makefile.{am,in}"],
+      "/System/Library/Frameworks/Carbon.framework/Headers/Carbon.h",
+      "#{MacOS.sdk_path}/System/Library/Frameworks/Carbon.framework/Headers/Carbon.h"
```

Comment:
Ah, I see it was already there. Would still be good to do so at some point but doesn't need to block this PR.